### ISSUE TITLE
Divide the links in sections

### ISSUE THREAD

**Issue by FedericoPonzi** [uhub/awesome-erlang#1]
This could improve readability.

**Comment by ccding** [comment]
I agree with you but this list was generated by a program which sorts repos by the number of stars. If you could help to divide them in sections, I am more than happy to accept your pull request. Check this repo https://github.com/uhub/awesome-actionscript-sorted as an example.

**Comment by crownedgrouse** [comment]
Hi, my project https://github.com/crownedgrouse/steam extract debian tags from Erlang project.  This could help sorting project by section.
